meson.build

```diff
@@ -64,31 +71,5 @@ if host_machine.system() != 'windows'
 endif
 thread_dep = dependency('threads')
 
-if host_machine.system() == 'linux'
-  libdrm_dep = dependency('libdrm', required : get_option('encoder'))
-  libvhal_dep = dependency('vhal-client',
-    required : get_option('encoder').enabled() or get_option('streamer').enabled())
-  libvpl_dep = dependency('vpl', required : get_option('encoder'))
-  libva_dep = dependency('libva', required : get_option('encoder'))
-
-  libavcodec_dep = dependency('libavcodec', required : get_option('encoder'))
-  libavfilter_dep = dependency('libavfilter', required : get_option('encoder'))
-  libavformat_dep = dependency('libavformat', required : get_option('encoder'))
-  libavutil_dep = dependency('libavutil', required : get_option('encoder'))
-
-  titansdk_dep = null_dep
-endif
-
-if host_machine.system() == 'windows'
-  winres_args = []
```

Comment:
These lines 83 to 87 should be removed in https://github.com/intel/cloud-streaming/pull/55/commits/20c6e270c5b24ae84bcce6987240dfc0c6d05303 not in https://github.com/intel/cloud-streaming/pull/55/commits/4232cfd9c08af84c4d513e8774160553898e0f1f

Reply:
This one is blocking for this PR. Others all look fine to me, just confirming.

Reply:
done